pyquil/api/_quantum_computer.py

```diff
@@ -654,7 +654,13 @@ def get_qc(name: str, *, as_qvm: bool = None, noisy: bool = None,
 
 
 @contextmanager
-def local_qvm() -> Iterator[Tuple[subprocess.Popen, subprocess.Popen]]:
+def local_runtime(
+        host: str = '127.0.0.1',
+        qvm_port: int = 5000,
+        quilc_port: int = 5555,
+        provide_http_server: bool = False,
```

Comment:
@ecpeterson what's the status of this in quilc? are we deprecating the HTTP server?

Reply:
It is RIP.